src/test/java/com/google/jenkins/plugins/storage/StdoutUploadStepTest.java

```diff
@@ -0,0 +1,117 @@
+package com.google.jenkins.plugins.storage;
+
+import static org.mockito.Matchers.isA;
+import static org.mockito.Mockito.when;
+
+import com.cloudbees.plugins.credentials.SystemCredentialsProvider;
+import com.google.api.client.googleapis.auth.oauth2.GoogleCredential;
+import com.google.api.services.storage.Storage;
+import com.google.api.services.storage.model.Bucket;
+import com.google.jenkins.plugins.credentials.oauth.AbstractGoogleRobotCredentialsDescriptor;
+import com.google.jenkins.plugins.credentials.oauth.GoogleOAuth2ScopeRequirement;
+import com.google.jenkins.plugins.credentials.oauth.GoogleRobotCredentials;
+import com.google.jenkins.plugins.util.MockExecutor;
+import com.google.jenkins.plugins.util.NotFoundException;
+import hudson.model.FreeStyleBuild;
+import hudson.model.FreeStyleProject;
+import hudson.model.TaskListener;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.jvnet.hudson.test.JenkinsRule;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+
+/** Tests for {@link StdoutUploadStep} */
+public class StdoutUploadStepTest {
+  @Rule public JenkinsRule jenkins = new JenkinsRule();
+
```

Comment:
Remove extra '\n' between instance variable declarations.

Reply:
Here and throughout.

Reply:
Done